rafthttp/stream.go

```diff
@@ -380,6 +400,18 @@ func (cr *streamReader) dial() (io.ReadCloser, error) {
 		return nil, err
 	case http.StatusOK:
 		return resp.Body, nil
+	case http.StatusNotFound:
+		b, err := ioutil.ReadAll(resp.Body)
```

Comment:
Two cases: 1. the stream type doesn't exist in current binary 2. the member has not been added into rafthttp